frontend/src/metabase/lib/redux/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from "./utils";
+export * from "./utils-ts";
```

Comment:
Converting the whole `utils.js` into TypeScript would be too much, so I only extract `createAction` which has a problem when using actions created from it in TypeScript files.